scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -395,7 +395,7 @@ <h4>Scout case analysis report</h4> - created on:&nbsp;<strong>{{report_created_
       No ACMG-classified variants available for this case
     {% endif %}
     {% if variants.classified_detailed and duplicated_variants|length == variants.classified_detailed|length %}
-      All ACMG-classified variants are already described in the previous views
+      All ACMG-classified variants are already described in the other panels
```

Comment:
Not previous any more, since it will be possibly found among the dismissed, which is the last panel